app/views/users/_form.html.haml

```diff
@@ -17,9 +17,9 @@
         = f.text_field :email, disabled: @user, class: 'small-12 medium-4'
     .grid-x
       .cell.small-12.medium-2
-        = f.label :upn
+        = f.label :uid
```

Comment:
Does uid is same as username? if so, uid label might be confusing to general user. making label read as Username, but in backend it can be uid, makes sense here.

Reply:
See also #570

Reply:
I chose to remove mentions of the UID because it is all behind the scene.